db/universities.ts

```diff
@@ -47,7 +56,7 @@ const getAll = async (): Promise<any[]> => {
 
 const getAllOrderByName = async (): Promise<any[]> => {
   try {
-    return db.select('id', 'name', 'emailSSU', 'emailUniversity')
+    return db.select('id', 'name')
```

Comment:
I understand the needs here, but this is used in the mail summary
it would be better to filter those in the designated controller

Reply:
The mail summary doesn't use `getAllOrderByName` but `getAll` and this one still select `emailSSU` and `emailUniversity`.